jarvis.go

```diff
@@ -2,23 +2,17 @@ package main
 
 import (
 	"github.com/gin-gonic/gin"
+	"github.com/magrathealabs/jarvis/domain/repositories"
 	"github.com/magrathealabs/jarvis/infrastructure/handlers"
 	"github.com/magrathealabs/jarvis/libs/env"
-	"github.com/magrathealabs/jarvis/libs/handler"
 )
 
-func usedHandlers() []handler.Handler {
-	return []handler.Handler{
-		handlers.NewRootHandler(),
-	}
-}
+var metricRepository repositories.MetricRepository
```

Comment:
Can you change this to constant?

Reply:
Não tem como mudar para constante, pois somente tipos primitivos podem ser constantes. Porém, no futuro isto será um método para instanciar o repositório (conectar ao graphite, etc)